Gems/Multiplayer/Code/Source/Debug/MultiplayerDebugAuditTrail.h

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/EntityBus.h>
+#include <AzCore/EBus/ScheduledEvent.h>
+#include <AzFramework/Entity/EntityDebugDisplayBus.h>
+#include <Multiplayer/Components/NetworkHierarchyRootComponent.h>
+
+namespace Multiplayer
+{
+    struct AuditTrailInput
+    {
+        AuditTrailInput(AuditCategory category, ClientInputId inputId, HostFrameId hostFrameId,
+            const AZStd::string name, const AZStd::vector<MultiplayerAuditingElement>&& children)
+            : m_category(category)
+            , m_inputId(inputId)
+            , m_hostFrameId(hostFrameId)
+            , m_name(name)
+            , m_children(children)
+        {
+        }
+
+        AuditCategory m_category;
+        ClientInputId m_inputId;
+        HostFrameId m_hostFrameId;
+        AZStd::string m_name;
+        AZStd::vector<MultiplayerAuditingElement> m_children;
+    };
+
+    //! Buffer size for ImGui Search bar in Audit Trail UI
+    const int AUDIT_SEARCH_BUFFER_SIZE = 1024;
```

Comment:
nit: Considering adding comments about how set, or implications for changing value.